src/app/page.tsx

```diff
@@ -3,10 +3,20 @@
 import { HeaderPage } from '@/Features/header';
 import { PromoCard } from '@/components/PromoCard';
 
+import { Footer } from '@/components/Footer';
+
 const Home = async () => {
-  const response = await fetch('http://localhost:3000/api/barbershops');
-  const jsonResponse = await response.json();
-  const numberOfBarbershops = jsonResponse.data;
+  const barbershopsInfoResponse = await fetch('http://localhost:3000/api/barbershopsInfo');
+  const barbershopsInfoJsonResponse = await barbershopsInfoResponse.json();
+  const numberOfBranches = barbershopsInfoJsonResponse.branchesTotal;
+  const numberOfCountries = barbershopsInfoJsonResponse.countriesTotal;
+  const numberOfCapitals = barbershopsInfoJsonResponse.capitalsTotal;
+
+  const footerListsResponse = await fetch('http://localhost:3000/api/footerLists');
+  const footerListsJsonResponse = await footerListsResponse.json();
+  const { footerMainList } = footerListsJsonResponse;
```

Comment:
check the comment above about destructuring.

Plus, I have one more improvement! Use Promise.all() for these 2 request. They do not depend on each other, so you can do it

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all#description

Reply:
ok, we will discuss it during the stand up